desktop/lib/store-kit/index.js

```diff
@@ -0,0 +1,50 @@
+var $ = require( 'nodobjc' );
+
+$.framework( 'Foundation' );
+$.framework( 'StoreKit' );
+
+function log() {
+	console.log.apply( null, [ 'StoreKit:' ].concat( [].slice.call( arguments ) ) );
+}
+
+function requestProducts() {
+	$.NSAutoreleasePool( 'alloc' )( 'init' );
```

Comment:
I'm pretty sure you need to stick this into a `var pool`, and then do `pool('release')` once you're done with all the objects that were created.